src/main/java/com/northconcepts/datapipeline/foundations/examples/pipeline/ShowColumnUniqueValuesStatistics.java

```diff
@@ -0,0 +1,41 @@
+package com.northconcepts.datapipeline.foundations.examples.pipeline;
+
+import com.northconcepts.datapipeline.foundations.file.LocalFile;
+import com.northconcepts.datapipeline.foundations.pipeline.Pipeline;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Column;
+import com.northconcepts.datapipeline.foundations.pipeline.dataset.Dataset;
+import com.northconcepts.datapipeline.foundations.pipeline.input.ExcelPipelineInput;
+import com.northconcepts.datapipeline.internal.lang.LongPointer;
+
+import java.util.Map;
+
+public class ShowColumnUniqueValuesStatistics {
```

Comment:
Let's rename to "ShowUniqueValuesInColumn`.

Reply:
done